config/confd/templates/env.tmpl

```diff
@@ -18,7 +18,8 @@ VPN_BASE_MANAGEMENT_PORT=20000
 VPN_API_PORT=30000
 
 VPN_CONNECT_INSTANCE_COUNT={{ if eq (getv "/resin/vpn/production") "true" }}0{{ else }}1{{ end }}
-VPN_CONNECT_PROXY_PORT=3128
+VPN_CONNECT_PROXY_PORT={{if exists "/resin/vpn/vpn-connect-proxy-port"}}"{{getv "/resin/vpn/vpn-connect-proxy-port"}}"{{else}}{{end}}
```

Comment:
As far as I can see 3128 is hard-coded in the [helm charts](https://github.com/balena-io/helm-charts/blob/master/balena-vpn/templates/service-nodeport.yaml#L16) so I don't think this needs to be parameterised.